applications/fastapi-bootcamp/values-idfdemo.yaml

```
@@ -0,0 +1,6 @@
+image:
+  pullPolicy: Always
+  tag: "tickets-DM-43967"
```

Recommend deleting to use the appVersion instead:

```suggestion
  tag: "tickets-DM-43967"
```

---

See above.